WordPress/WordPressTest/Activity/ActivityListViewModelTests.swift

```diff
@@ -20,7 +20,7 @@ class ActivityListViewModelTests: XCTestCase {
     // Check if `loadMore` dispatchs the correct action and params
     //
     func testLoadMore() {
-        let jetpackSiteRef = JetpackSiteRef.mock(siteID: 0, username: "")
+        let jetpackSiteRef = JetpackSiteRef.mock(siteID: 0, username: "", homeURL: "")
```

Comment:
Since the `homeURL` is not important for these tests we can just suppress this argument?